modules/core/src/lib/deck.js

```diff
@@ -179,6 +179,8 @@ export default class Deck {
     this._onRenderFrame = this._onRenderFrame.bind(this);
     this._onViewStateChange = this._onViewStateChange.bind(this);
     this._onInteractionStateChange = this._onInteractionStateChange.bind(this);
+    this._onWebGLContextLost = this._onWebGLContextLost.bind(this);
+    this._onWebGLContextRestored = this._onWebGLContextRestored.bind(this);
```

Comment:
It's excessive to bind these methods given that they are only referenced once. These method names are very long and cannot be minified. The existing code that binds other methods can probably use a clean up too.